CMakeLists.txt

```diff
@@ -185,11 +185,23 @@ endif()
 if (CMAKE_CONFIGURATION_TYPES)
   foreach (build_type IN LISTS CMAKE_CONFIGURATION_TYPES)
     execute_process(COMMAND "${CMAKE_COMMAND}" -E make_directory "${CMAKE_CURRENT_BINARY_DIR}/${build_type}/include")
-    execute_process(COMMAND "${CMAKE_COMMAND}" -E ${LINK_COMMAND} "${CMAKE_CURRENT_SOURCE_DIR}/runtime" "${CMAKE_CURRENT_BINARY_DIR}/${build_type}/include/runtime")
+    if (CMAKE_HOST_WIN32)
+      file(TO_NATIVE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/runtime" _srcDir)
+      file(TO_NATIVE_PATH "${CMAKE_CURRENT_BINARY_DIR}/${build_type}/include/runtime" _dstDir)
+      execute_process(COMMAND cmd.exe /c mklink /J "${_dstDir}" "${_srcDir}")
+    else()
+      execute_process(COMMAND "${CMAKE_COMMAND}" -E ${LINK_COMMAND} "${CMAKE_CURRENT_SOURCE_DIR}/runtime" "${CMAKE_CURRENT_BINARY_DIR}/${build_type}/include/runtime")
```

Comment:
If we're going to use a junction instead of a hard/softlink, then we should get rid of all the logic above that sets LINK_COMMAND, including the cache variable.

Reply:
`LINK_COMMAND` is still effective on *nix if someone wanted a hard link over symlink there.